quam/components/channels.py

```diff
@@ -396,7 +401,13 @@ def play(
             )
 
         if amplitude_scale is not None:
-            if not isinstance(amplitude_scale, _PulseAmp):
+            if isinstance(amplitude_scale, _PulseAmp):
+                warnings.warn(
+                    "Setting amplitude_scale=amp(...) is deprecated, please "
+                    "pass a float or list of floats instead",
+                    DeprecationWarning,
+                )
+            else:
                 amplitude_scale = amp(amplitude_scale)
```

Comment:
If it's a list of floats, you need to do:
`amplitude_scale = amp(*amplitude_scale)`

Reply:
Nice catch! Fixed